src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring/Generated/Models/ApplicationPublishObject.cs

```diff
@@ -32,13 +32,10 @@ public ApplicationPublishObject()
         /// <param name="versionId">The version ID to publish.</param>
         /// <param name="isStaging">Indicates if the staging slot should be
         /// used, instead of the Production one.</param>
-        /// <param name="region">The target region that the application is
-        /// published to.</param>
-        public ApplicationPublishObject(string versionId = default(string), bool? isStaging = default(bool?), string region = default(string))
+        public ApplicationPublishObject(string versionId = default(string), bool? isStaging = default(bool?))
         {
             VersionId = versionId;
             IsStaging = isStaging;
-            Region = region;
```

Comment:
This (I guess) should be considered as a breaking change. Thus might require a major version bump.